optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -1,5 +1,7 @@
+
 [[quickStart]]
-= Quick Start
+= Getting Started: A Cloud Balancing Demonstration
+//Christy suggested that this isn't really a "quickstart". I don't know how you feel about renaming a chapter?
```

Comment:
Hey Geoffrey, I asked Christy to clarify this, and the basic idea is that we (as in, CCS as a whole), are working towards more clear and consistent terminology and a standardization of how we use certain terms. "Quickstart", in our terminology, would refer to something that includes installation information to actually get a product or program up and running, whereas "Getting Started" focuses more on tutorials and how to actually use the product once you have already installed it and have it up and running.